### PR TITLE
Reformat and fix guild_items_points.sql

### PR DESCRIPTION
**Pull request by davismj**
- [x] Fishing
- [x] Woodworking
- [x] Blacksmithing
- [x] Goldsmithing
- [x] Clothcraft
- [x] Leathercraft
- [x] Bonecraft
- [x] Alchemy
- [x] Cooking

Fixes https://github.com/DarkstarProject/darkstar/issues/5636

### CODE REVIEW

File: sql/guild_item_points.sql [hunk 1]
```diff
@@ -23,1280 +23,1055 @@ CREATE TABLE `guild_item_points` (
...
-INSERT INTO `guild_item_points` VALUES ('5', '13919', '9', '1080', '5040', '6');
-INSERT INTO `guild_item_points` VALUES ('5', '13960', '6', '3547', '6400', '1');
-INSERT INTO `guild_item_points` VALUES ('5', '13992', '7', '1912', '5680', '1');
-INSERT INTO `guild_item_points` VALUES ('5', '14057', '8', '3600', '6480', '6');
-INSERT INTO `guild_item_points` VALUES ('5', '14068', '0', '63', '1440', '2');
-INSERT INTO `guild_item_points` VALUES ('5', '14069', '0', '71', '1440', '2');
-INSERT INTO `guild_item_points` VALUES ('5', '14070', '1', '360', '2880', '7');
-INSERT INTO `guild_item_points` VALUES ('5', '14071', '1', '986', '2280', '7');
-INSERT INTO `guild_item_points` VALUES ('5', '14072', '2', '519', '3520', '2');
-INSERT INTO `guild_it
```
Comment (ShelbyZ) [inline sql/guild_item_points.sql]: What value should the '/' be?

You can find these errors by looking at the two build systems (specifically travis-ci for sql changes):

```
Importing guild_item_points.sql into the database...ERROR 1064 (42000) at line 73: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '/, 800, 3)' at line 1
```

**Comment by zynjec** [comment]
`Importing guild_item_points.sql into the database...ERROR 1062 (23000) at line 291: Duplicate entry '2-0-0' for key 'PRIMARY'`

**Comment by davismj** [comment]
@zynjec yea still WIP, need to fill out the item ids after i fill out point items

File: sql/guild_item_points.sql [hunk 2]
```diff
@@ -23,1280 +23,1482 @@ CREATE TABLE `guild_item_points` (
...
```
Comment (davismj) [inline sql/guild_item_points.sql]: Someone pointed out this is https://ffxiclopedia.fandom.com/wiki/Argute_Belt

**Comment by davismj** [comment]
This PR is ready for review.

File: sql/guild_item_points.sql [hunk 3]
```diff
@@ -23,1280 +23,913 @@ CREATE TABLE `guild_item_points` (
 ) ENGINE=MyISAM DEFAULT CHARSET=utf8;
 
 -- ----------------------------
--- Records 
+-- Records
 -- ----------------------------
-INSERT INTO `guild_item_points` VALUES ('0', '2850', '9', '2850', '6240', '3');
-INSERT INTO `guild_item_points` VALUES ('0', '4288', '6', '1200', '5040', '2');
```
Comment (TeoTwawki) [inline sql/guild_item_points.sql]: for acts of bravery in the ongoing war against quoted non strings, I present you with this medal :medal_military:

Comment (davismj) [reply sql/guild_item_points.sql]: i rewrote this file from scratch :(

Comment (TeoTwawki) [reply sql/guild_item_points.sql]: ok fine, 2 medals. :medal_military: :medal_sports:

**Pull request merged**